src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getName()
+                + ", area: " + calculateArea() + " sq.units, "
+                + "radius: " + getRadius() + " units, "

```

Comment:
```suggestion
                + "radius: " + radius + " units, "
```
you have direct access to this field inside the class - no need to use a method to get it